src/Codata/Delay/Properties.agda

```diff
@@ -48,3 +49,64 @@ module _ {a b c} {A : Set a} {B : Set b} {C : Set c} where
  map-unfold-fusion f n s with n s
  ... | Sum.inj₁ s′ = later λ where .force → map-unfold-fusion f n s′
  ... | Sum.inj₂ b  = now Eq.refl
+
+
+⇓-unique : ∀ {a} → {A : Set a} →
+           {d : Delay A ∞} →
+           (d⇓₁ : d ⇓) → (d⇓₂ : d ⇓) →
+           d⇓₁ ≡ d⇓₂
+⇓-unique {d = now s} (now s) (now s) = Eq.refl
+⇓-unique {d = later d'} (later l) (later r) =
+  Eq.cong later (⇓-unique {d = force d'} l r)
+
+
+bind⇓-injₗ : ∀ {a} {A B : Set a} →
```

Comment:
I believe this should be called `bind⁻₁`. Cf. the [style guide](https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md#functions-and-relations-over-specific-datatypes)

Same notation for the following lemmas.